ui/components/app/confirm/info/row/alert-row/alert-row.tsx

```diff
@@ -74,7 +74,7 @@ export const AlertRow = ({
     <>
       {alertModalVisible && (
         <MultipleAlertModal
-          alertKey={alertKey}
+          alertKey={fieldAlerts[0].key}
```

Comment:
I am bit confused here, is it ok to hardcode key to always alert at index `0` ?

Reply:
It is safe, since the modal isn't visible unless the alert is clicked, which is only shown if there are field alerts.

Reply:
I like that, but perhaps we could do that in a separate PR to limit the scope of this big one?